src/libraries/System.Speech/src/Internal/ObjectToken/ObjectTokenCategory.cs

```diff
@@ -21,7 +21,12 @@ protected ObjectTokenCategory(string keyId, RegistryDataKey key)
         internal static ObjectTokenCategory Create(string sCategoryId)
         {
             RegistryDataKey key = RegistryDataKey.Open(sCategoryId, true);
```

Comment:
I assume this doesn't throw if the key isn't there -- it ends up calling into the COM interface so I guess that's the case